pkg/kubectl/cmd/set/helper.go

```diff
@@ -139,7 +139,14 @@ func CalculatePatch(patch *Patch, encoder runtime.Encoder, mutateFn patchFn) boo
 		return false
 	}
 
-	// TODO: should be via New
+	// Create the versioned struct from the type defined in the restmapping
+	// (which is the API version we'll be submitting the patch to)
+	_, err := legacyscheme.Scheme.New(patch.Info.Mapping.GroupVersionKind)
```

Comment:
what is the purpose of this call? the result isn't used...

Reply:
Do your know this Todo mean?